scipy/optimize/tests/test_optimize.py

```diff
@@ -2455,21 +2456,20 @@ def test_powell_output():
 
 
 class TestRosen:
+    @make_skip_xp_backends("rosen")
     def test_rosen(self, xp):
         # integer input should be promoted to the default floating type
         x = xp.asarray([1, 1, 1])
         xp_assert_equal(optimize.rosen(x),
                         xp.asarray(0.))
 
-    @skip_xp_backends('jax.numpy',
-                      reason="JAX arrays do not support item assignment")
+    @make_skip_xp_backends("rosen_der")
     def test_rosen_der(self, xp):
         x = xp.asarray([1, 1, 1, 1])
         xp_assert_equal(optimize.rosen_der(x),
                         xp.zeros_like(x, dtype=xp.asarray(1.).dtype))
 
-    @skip_xp_backends('jax.numpy',
-                      reason="JAX arrays do not support item assignment")
+    @make_skip_xp_backends("rosen_hess_prod")
     def test_hess_prod(self, xp):
```

Comment:
The change here associated with the other suggestion might be:
```suggestion
    @make_skip_xp_backends("rosen_hess")
    def test_hess(self, xp):
```